blueprints/app/files/testem.js

```diff
@@ -11,10 +11,12 @@ module.exports = {
     Chrome: {
       ci: [
         // --no-sandbox is needed when running Chrome inside a container
-        process.env.CI ? '--no-sandbox' : null,
-
-        '--disable-gpu',
+        process.env.TRAVIS ? '--no-sandbox' : null,
```

Comment:
was the change from `CI` to `TRAVIS` intended?

Reply:
good catch, we want CI here